langs/kotlin.go

```diff
@@ -317,7 +317,10 @@ package com.fn.example
 
 fun hello(input: String) = when {
     input.isEmpty() -> ("Hello, world!")
-        else -> ("Hello, ${input}")
+        else -> {
+          println("Inside Kotlin Hello World function")
```

Comment:
@srpurani - For the Kotlin one, can you keep the print statement outside the if/else block?

Reply:
Done.